### ISSUE TITLE
Time machine: generated CSS is not exported

### ISSUE THREAD

**Issue by jorith88** [dotCMS/core#8028]
When I use /DOTLESS/... or /DOTSASS/... to generate CSS from SASS/LESS files, the generated CSS is not exported by the Time machine. 

This means that it's not possible to export the generated HTML to an Apache webserver when using the built in SASS/LESS support.

**Comment by koenpeters** [comment]
We had the same issue with our minifier plugin. What we ended up doing was instead of using a servlet to put the minified data in the cache actually store the generated minified files in the dotCMS virtual file system so they would be included in a time machine export. That works pretty well but does have the downside of having generated files in the dotCMS file view and having old versions there that are never removed. That's why we also added a cron job to clear the generated files every day and have dotCMS generate new ones once a day at first request.
